bin/setup

```diff
@@ -3,4 +3,4 @@
 # A setup script for development environment of Granite
 
 cp spec/support/database.yml.example spec/support/database.yml
-psql -c 'create database granite;' -U granite
+psql template1 -c 'create database granite;' -U granite
```

Comment:
This command will fail if `database` and `username` options in [`spec/support/database.yml.example`])https://github.com/toptal/granite/blob/master/spec/support/database.yml.example#L3) are configured to use a different database and username, not to say password. Wondering if it's possible to take options from that file to create a test database.

Anyway,
```
createdb granite
```
works quite fine on my machine.

Reply:
I forgot about `createdb`. let's use the proper command.